eng/configurations/targetgroups.props

```diff
@@ -42,15 +42,15 @@
       <PackageTargetRuntimeSuffix>aot</PackageTargetRuntimeSuffix>
       <UWPCompatible>true</UWPCompatible>
       <Imports>uap10.0.16299aot</Imports>
-      <CompatibleWith>uapvnext;netstandard2.0</CompatibleWith>
+      <CompatibleWith>uapvnext;netstandard2.1</CompatibleWith>
```

Comment:
Why not 2.0?

Reply:
This change has nothing to do with the package, the changes on this file are only so that we can have netstandard2.1 configurations.